fancy/analysis/analysis.py

```diff
@@ -612,7 +613,7 @@ def _simulate_deflections(self, kappas, Nrand=100, path_to_lens=None):
         This then returns the simulated arrival directions and detected energies
         at Earth after deflection.
 
-        Composition is accounted for by using the rigidities instead of energies in the lensing process.
+        Composition is accounted for by using thy instead of energies in the lensing process.
```

Comment:
typo?

Reply:
Looks like a typo to me, I can fix this with a new commit